grants_tagger/tag_grants.py

```diff
@@ -31,15 +25,13 @@ def yield_tagged_grants(grants, scibert_path, tfidf_svm_path, label_binarizer_pa
     Returns
         tagged_grant: dict with keys
             Grant ID, Reference, Grant No
-            ScienceCategory#{1..9}
-            DiseaseCategory#{1..9)
+            Tag #{1..10}
     """
     grants_texts = [g["title"]+g["synopsis"] for g in grants]
-    grants_tags = predict_tags(
+    tags = predict_tags(
         grants_texts,
         threshold=threshold,
-        scibert_path=scibert_path,
-        tfidf_svm_path=tfidf_svm_path,
+        model_path=model_path,
         label_binarizer_path=label_binarizer_path
     )
     for grant, tags in zip(grants, grants_tags):
```

Comment:
something needs to change here since `grant_tags` doesn't exist and you are also rewriting `tags`

Reply:
oups, better add a test that can catch this error as well

Reply:
f7d9533